src/components/LoginPage/Slide5.js

```diff
@@ -0,0 +1,125 @@
+};
+
+const CaptionBoxSlide5 = ({ viewBoxWidth = 60, viewBoxHeight = 40, ...rest }) => {
+    return <Caption viewBox={`0 0 ${viewBoxWidth} ${viewBoxHeight}`} {...rest} />;
+};
+
+export default class Slide5 extends Component {
+    render() {
+      return (
+        <Slide>
+            <H3>Frequently used notes<br />get bigger</H3>
+            <Hint className='hint-old-notes'>
+              <CaptionBoxSlide5 viewBoxWidth="160" viewBoxHeight="40" left="3%" top="10%" width="94%" fontWeight="700">
+                <CaptionTextSlide5 x="6" y="18" fill="#E8E9F8" text="OLD UNUSED NOTES" />
+                <CaptionTextSlide5 x="36" y="38" fill="#E8E9F8" text="ARE" />
+              <CaptionTextSlide5 x="71" y="38" fill="#96FFFF" text="SMALL" />
+            </CaptionBoxSlide5>
+              <ArrowDown src={arrow} />
+            </Hint>
+            <Figure>
+            <Img src={imageMindmap} shadow />
+              <CaptionBoxSlide5 viewBoxWidth="100" viewBoxHeight="21" left="34%" top="21%" width="15.5%" fontWeight="700">
+              <CaptionTextSlide5 x="6" y="18" fill="#131C3E" text="NeuralNotes" />
+            </CaptionBoxSlide5>
+              <CaptionBoxSlide5 viewBoxWidth="45" left="52.5%" top="48%" width="9.5%" fontWeight="700">
+                <CaptionTextSlide5 y="16" text="Work" />
+            </CaptionBoxSlide5>
+              <CaptionBoxSlide5 left="19.5%" top="51%" width="13.5%" fontWeight="700">
+              <CaptionTextSlide5 y="16" text="Current" />
+                <CaptionTextSlide5 y="32" text="Project" />
+            </CaptionBoxSlide5>
+              <CaptionBoxSlide5 left="14.5%" top="26%" width="7.5%">
+                <CaptionTextSlide5 y="14" text="Personal" />
+                <CaptionTextSlide5 x="20" y="32" text="life" />
+            </CaptionBoxSlide5>
+              <CaptionBoxSlide5 left="61%" top="25.5%" width="7.5%">
+              <CaptionTextSlide5 x="12" y="14" text="Old" />
+                <CaptionTextSlide5 y="32" text="project" />
+              </CaptionBoxSlide5>
+              <CaptionBoxSlide5 viewBoxWidth="70" left="81%" top="35%" width="8%">
+                <CaptionTextSlide5 y="14" text="Forgotten" />
+                <CaptionTextSlide5 x="10" y="32" text="project" />
+              </CaptionBoxSlide5>
+              <CaptionBoxSlide5 viewBoxWidth="82" left="79%" top="60%" width="10%">
+              <CaptionTextSlide5 y="14" text="Abandoned" />
+                <CaptionTextSlide5 x="17" y="32" text="project" />
+              </CaptionBoxSlide5>
+            </Figure>
+          <Hint className='hint-recent-notes'>
+            <ArrowUp src={arrow} />
+            <CaptionBoxSlide5 viewBoxWidth="170" viewBoxHeight="40" left="7%" top="35%" width="86%" fontWeight="700">
+              <CaptionTextSlide5 x="3" y="18" fill="#E8E9F8" text="RECENT, FREQUENTLY" />
+              <CaptionTextSlide5 x="3" y="38" fill="#E8E9F8" text="USED NOTES ARE" />
```

Comment:
Color `#E8E9F8` is repeated 5 times. Please create a `colors.js` file and store all the colors there, and in slides use only variables. Or use the existing `src/colors.js` file: https://github.com/DarthVanger/NeuralNotes/blob/master/src/colors.js

I'm just not sure if we're going to share the colors between the login page and the app itself, so it might be more handy for `LoginPage` to have it's own `LoginPage/colors.js`.

Reply:
Done in commit [c85218d](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/c85218d4ff8d3b1e710eca88506e56fe1f27697a)